### PR TITLE
Create gui demo repo for cargo application

### PR DESCRIPTION
**Pull request by GoodluckH**
Frontend will be implemented in Vue.

### CODE REVIEW

File: src/main.tf [hunk 1]
```diff
@@ -142,6 +142,13 @@ locals {
       webhooks         = try(local.webhooks["services"], {})
       repository_files = local.rust_default.files
```
Comment (owlot) [inline src/main.tf:143]: A space is missing here, can you run `terraform fmt` and push the changes? That should fix the failing check.

**Review by owlot — CHANGES_REQUESTED** [review]

File: src/main.tf (hunk 1 above)
Comment (GoodluckH) [reply src/main.tf:143]: I did but nothing happened

Comment (ServiceDog) [reply src/main.tf:143]: cd into the src/ directory first?

Comment (GoodluckH) [reply src/main.tf:143]: that worked! thank you.

**Review by owlot — APPROVED** [review]

**Review by ServiceDog — APPROVED** [review]

**Pull request merged**